pkgs/tools/system/datefudge/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchgit, fetchpatch }:
+{ stdenv, lib, fetchgit, fetchpatch, makeWrapper, coreutils ? null }:
```

Comment:
I would prefer if we just always pass in coreutils if this is possible.

Reply:
On build system -- yes, coreutils is part of stdenv. On host system -- not necessary.